Assets/Scripts/ProjectileHazardScript.cs

```diff
@@ -15,25 +16,25 @@ public class ProjectileHazardScript : MonoBehaviour
     void Start()
     {
         isMoving = false;
+        playerRocket = GameObject.Find("Rocket");
     }
 
     // Update is called once per frame
-    void Update()
+    private void Update()
     {
         Collider[] hitColliders = Physics.OverlapSphere(transform.position,pullRadius);
         
         foreach (var currentCollider in hitColliders)
         {
-            if (!isMoving){
+            if (isMoving){
+                GetComponent<Rigidbody>().AddForce(initialRocketPosition * strengthOfAttraction * Time.deltaTime);                
+            } else
                 if (currentCollider.tag == "Player")
```

Comment:
Instead of hardcoding a string here let's use a constant. The constant can be called `PLAYER_TAG`. You can declare it in the global scope like this:

```
public class ProjectileHazardScript : MonoBehaviour
{
    private const string PLAYER_TAG = "Player";
```